### ISSUE TITLE
MIDL not supported?

### ISSUE THREAD

**Issue by manu-st** [playscale/stashed.io#8]
I'm compiling a project using some IDL files for which I generate some C++ code to be compiled. When stashed is enabled, I'm getting the following error:

```
"C:\Dev\src\project.sln" (Build target) (1) ->
       "C:\Dev\src\sub\sub.vcxproj" (default target) (7) ->
       (Midl target) ->
         sub.idl(1): error MIDL2183: unexpected end of file found [C:\Dev\src\sub.vcxproj]
```

**Comment by jsmouret** [comment]
Thanks, I'll take a look

**Comment by jsmouret** [comment]
Fixed in version 1.4.1
I'll leave the issue opened for a while in case you find something related.

Thanks!

**Comment by manu-st** [comment]
Thanks. It works!
